cmd/commands/cli/command.go

```diff
@@ -431,6 +433,29 @@ func (c *cliApp) natStatus() (err error) {
 	} else {
 		clio.Infof("NAT traversal status: %q (error: %q)\n", status.Status, status.Error)
 	}
+
+	connStatus, err := c.tequilapi.ConnectionStatus()
+	if err != nil {
+		clio.Warn(err)
+		return
+	}
+
+	if connStatus.Status == statusNotConnected {
```

Comment:
```suggestion
	if connStatus.Status != statusNotConnected {
	  return nil
	}
```

Then you can unnest the contents